test/common-utils/scenarios.json

```diff
@@ -41,14 +41,42 @@
             "common-utils": {}
         }
     },
+    "alma-minimal-8": {
+        "image": "almalinux:8-minimal",
+        "remoteUser": "devcontainer",
+        "features": {
+            "common-utils": {}
```

Comment:
This is a failing test in the `common-utils` feature, not caused by the git feature. I can certainly take a look, but I think the fix for that test would be in a separate PR.

Reply:
The other reason for the failure is I've got a test on "minimal" images of AlmaLinux and RedHat Linux where yum is not installed, but microdnf is installed. It can be nice to use those as images for the basis of images/containers because you end up with a much smaller image in the end.

I see three choices here:
1. I remove the "minimal" image tests from the PR -- personally, I'd like to keep them because I want to know I can start with a minimal build.
2. I don't test with the `common-utils` feature included and rather update the "git" feature to install, if needed, more packages to get git built when needed.
3. I help fix the `common-utils` feature and add tests to it for the "minimal" images like I have for git.

I'm quite willing to file a ticket reflecting this problem, get it fixed and submit a PR against the ticket... basically taking approach #3.

Reply:
Appreciate your valuable thoughts and contributions  🙌 

> I'm quite willing to file a ticket reflecting this problem, get it fixed and submit a PR against the ticket... basically taking approach https://github.com/devcontainers/features/pull/3.

Like this approach, can we also remove the failing test scenarios this PR adds? 

>  personally, I'd like to keep them because I want to know I can start with a minimal build.

I understand your ask here, however, a failing build is not great and it stops the action runs, hence we will also get a ❌ for test runs. This can be misleading and we won't be able to check the complete health of `common-utils` Feature. Hence, I'd prefer if we remove the tests, and add them back with the fix PR. 

Let me know if you think otherwise, thanks!

Reply:
I've submitted a PR, https://github.com/devcontainers/features/pull/811, to fix the common-utils issue, allowing the `common-utils` feature to work with minimal os image. That will fix the failing tests in this PR. Once https://github.com/devcontainers/features/pull/811 is merged, there should be no failing test scenarios in this PR.

I can remove the test scenarios from this PR and file a new PR later to add them back after my `common-utils` PR is merged.